README.rst

```diff
@@ -36,3 +36,41 @@ normally used in B physics is also provided.
    - `iminuit <http://iminuit.readthedocs.org/>`_
    - `numpy <http://www.numpy.org/>`_
    - `matplotlib <http://matplotlib.org/>`_ (optional, for plotting)
+
+Development
+-----------
+
+Contributions to probfit are welcome. You should fork the repository,
+create a branch in your fork, and then `open a pull request 
+<https://github.com/iminuit/probfit/pulls>`_.
+
+Developing probfit requires a few dependencies, in addition to those required
+for users. The following commands should create a suitable environment,
+assuming you've cloned your fork of probfit and are in the repository root.
+(You may wish to work inside a virtual environment to isolate these packages
+from your system install.)
+
+.. code-block:: shell
+
+    $ pip install cython pytest pytest-mpl pylint flake8 sphinx sphinx_rtd_theme
+    $ make build
+
+Installing `Cython <http://cython.org/>`_ will allow you to build the C
+extensions. `Pylint <https://www.pylint.org/>`_ and `flake8 
+<https://pypi.python.org/pypi/flake8>`_ are used for linting, `pytest 
+<http://doc.pytest.org/en/latest/>`_ for testing, and `Sphinx 
+<http://www.sphinx-doc.org/en/1.4.8/>`_ for generating the HTML documentation.
```

Comment:
You link to http://www.sphinx-doc.org/en/1.4.8/
I'd suggest to link to just http://www.sphinx-doc.org/